DatadogSessionReplay/Sources/Recorder/Recorder.swift

```diff
@@ -117,6 +123,11 @@ public class Recorder: Recording {
             }
             let touchSnapshot = touchSnapshotProducer.takeSnapshot(context: recorderContext)
             snapshotProcessor.process(viewTreeSnapshot: viewTreeSnapshot, touchSnapshot: touchSnapshot)
+
+            resourceProcessor?.process(
+                resources: viewTreeSnapshot.resources,
+                context: .init(recorderContext.applicationID)
+            )
```

Comment:
when there are `UIImage` which are not loaded yet, how are these handled?

in short - when images are loaded async.

Reply:
This is only processing images that showed up on the screen (and met recording criteria).

Right now we don't support images downloaded in the runtime, but we might in the future.

Reply:
How do we make sure that image is loaded? Just curious on triggering logic, if you know the line of code, would be great.